src/plugins/multimedia/multimedia.js

```diff
@@ -679,7 +679,7 @@ $document.on( initializedEvent, selector, function( event ) {
 			}
 
 			// finally lets load safely
-			return Modernizr.load( {
+			Modernizr.load( {
```

Comment:
There might also be value in implementing a failure/timeout check of sorts in case the YouTube API request fails. Like if YouTube is blocked in a corporate network. That would prevent other plugins (like the do action plugin - which GCWeb's page feedback widget depends on) from breaking on pages that contain YouTube videos in those kinds of networks.

Reply:
wb.ready() is now called on the onReady event. Works fine on unblocked networks.
on blocked networks, this solution works too, but there is a big delay in loading the YT api with Modernizr